CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# v1.29.0 Release - 07/29/2021
+Note: This release comes with a new config version, `v2beta20`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Skaffold master -> main [#6263](https://github.com/GoogleContainerTools/skaffold/pull/6263)
+* Add state to event handler to track `logrus` output events [#6272](https://github.com/GoogleContainerTools/skaffold/pull/6272)
+* Remove skaffoldLogs endpoint, consolidate into events endpoint [#6296](https://github.com/GoogleContainerTools/skaffold/pull/6296)
+* fix: kubeContext override via flag [#6331](https://github.com/GoogleContainerTools/skaffold/pull/6331)
+* Add ContainerTracker for tracking deployed docker containers [#6266](https://github.com/GoogleContainerTools/skaffold/pull/6266)
+* [v2] Enable `skaffold dev` [#6261](https://github.com/GoogleContainerTools/skaffold/pull/6261)
+* Added developer journey tutorial [#6201](https://github.com/GoogleContainerTools/skaffold/pull/6201)
+
+New Features:
+* Implement port forwarding for Docker deployer [#6303](https://github.com/GoogleContainerTools/skaffold/pull/6303)
+* Implement Logger for Docker Deployer [#6265](https://github.com/GoogleContainerTools/skaffold/pull/6265)
+* [v2] Add renderer manifests as dependencies  [#6277](https://github.com/GoogleContainerTools/skaffold/pull/6277)
+* [v2] Enable the helm in `.manifests.generate` (bypass kpt fn) [#6270](https://github.com/GoogleContainerTools/skaffold/pull/
+6270)
+* [v2] Set v2 default renderer and deployer config. [#6252](https://github.com/GoogleContainerTools/skaffold/pull/6252)
+* [v2] Add a new deployer kptV2. [#6235](https://github.com/GoogleContainerTools/skaffold/pull/6235)
```

Comment:
Can we remove this from notes. Currently users can't try this.

Reply:
Done